index.html

```diff
@@ -21,11 +21,11 @@
     <header id="main-header">
       <div id="nav-overlay" class="overlay">
         <div class="overlay-content">
-          <a href="#">Welcome</a>
-          <a href="#">Menu</a>
-          <a href="#">Reservations</a>
-          <a href="#">News</a>
-          <a href="#">Contact</a>
+          <a href="#hero">Welcome</a>
+          <a href="assets/pdf/Gustoso-menu.pdf">Menu</a>
```

Comment:
I am not 100% sure whether this will link correctly on the live website. I think it needs to be an absolute link.